CHANGELOG.md

```diff
@@ -8,10 +8,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+- **CUMULUS-1787**
+  - Added `collections/active` endpoint for returning collections with active granules in `@cumulus/api`
```

Comment:
Is this intentional given the PR is for 1799?

Reply:
Yeah I moved it because it was sitting under the header for release 1.21.0, but this work was merged after that release. The note was in the wrong place because the work was started before the release though